newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -134,46 +158,51 @@ class EventContainer extends Component<EventsContainerProps, {||}> {
         ref={container => (this._container = container)}
         onClick={this.props.onEventClick}
         onContextMenu={this._onEventContextMenu}
-        style={getEventContainerStyle(this.props.windowWidth)}
       >
         {EventComponent && (
-          <EventComponent
-            project={project}
-            scope={scope}
-            event={event}
-            globalObjectsContainer={this.props.globalObjectsContainer}
-            objectsContainer={this.props.objectsContainer}
-            selected={isEventSelected(this.props.selection, event)}
-            selection={this.props.selection}
-            leftIndentWidth={this.props.leftIndentWidth}
-            onUpdate={this._onEventUpdated}
-            onAddNewInstruction={this.props.onAddNewInstruction}
-            onPasteInstructions={this.props.onPasteInstructions}
-            onMoveToInstruction={this.props.onMoveToInstruction}
-            onMoveToInstructionsList={this.props.onMoveToInstructionsList}
-            onInstructionClick={this.props.onInstructionClick}
-            onInstructionDoubleClick={this.props.onInstructionDoubleClick}
-            onInstructionContextMenu={this.props.onInstructionContextMenu}
-            onAddInstructionContextMenu={this.props.onAddInstructionContextMenu}
-            onParameterClick={this.props.onParameterClick}
-            onOpenExternalEvents={this.props.onOpenExternalEvents}
-            onOpenLayout={this.props.onOpenLayout}
-            disabled={
-              disabled /* Use disabled (not event.disabled) as it is true if a parent event is disabled*/
-            }
-            renderObjectThumbnail={this.props.renderObjectThumbnail}
-            screenType={this.props.screenType}
-            windowWidth={this.props.windowWidth}
-            eventsSheetHeight={this.props.eventsSheetHeight}
-          />
+          <div style={styles.eventComponentContainer}>
+            {this.props.connectDragSource(
+              <div style={styles.handle} className={handle} />
```

Comment:
Styles is defined by both the style props and the className props. The classname was used because it was the only way to customize the rst handle but I think you can use only the style props now.

Reply:
It looks like there is some logic linked with the classname (the color and size of the handle will change when there is a search for example).
I will move the styles to the move-handle css instead!